resources/language/resource.language.nl_nl/strings.po

```diff
@@ -0,0 +1,166 @@
+# Service Up Next language file
+# Addon Name: Up Next
+# Addon id: service.upnext
+# Addon Provider: im85288
+msgid ""
+msgstr ""
+"Project-Id-Version: service.upnext\n"
+"Report-Msgid-Bugs-To: http://trac.xbmc.org/\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2018-02-26 13:31+0000\n"
+"Last-Translator: dagwieers\n"
+"Language-Team: Dutch\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: nl\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7\n"
+
+msgctxt "#30001"
+msgid "The number of seconds before the end to show the notification"
+msgstr "Aantal seconden voor het einde om meldingen te tonen"
```

Comment:
Er is maar één melding, dus beter enkelvoud:
Het aantal seconden voor het einde om de melding te tonen

Reply:
Wel, ik wilde het korter maken, en het meervoud is korter dan het enkelvoud :laughing: 
Maar het gaat hier over toekomstige meldingen.

Reply:
Eventueel nog korter zodat het mooi past in de GUI: Eindmarge voor meldingen (in seconden)